client/src/modules/cohort/index.jsx

```diff
@@ -406,6 +533,11 @@ Cohort.propTypes = {
   archivedLists: PropTypes.objectOf(PropTypes.object),
   cohortDetails: PropTypes.shape({
     description: PropTypes.string,
+    externalAction: PropTypes.shape({
```

Comment:
I think it would be nice to extract this Prop type to common directory because I suppose that we gonna use this many times across the project